### PR TITLE
Add new program state MENTOR_CONFIRMATION to dashboard

### LINKED ISSUE #62: Add new program state into the change state UI
**Is your feature request related to a problem? Please describe.**
As a new program state's introduced for mentees to select a mentor of their choice, the change state UI needs to be updated accordingly.

**Describe the solution you'd like**
Add the new program state into the existing UI

**Additional context**
Information about the new program state could be found in this hive thread.
[Adding a new program state for mentees to confirm the mentor](https://sef.discourse.group/t/adding-a-new-program-state-for-mentees-to-confirm-the-mentor/216)

### PR DESCRIPTION
**Pull request by Gimhan-minion**
## Purpose
The purpose of this PR is to fix #62 

## Goals
- To edit the change state UI in order to add the new program state.

## Approach
- Edited the code accordingly and added the new program state `MENTOR_CONFIRMATION`

## Screenshots
Screenshot from 2020-12-21 03-11-31

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: src/scenes/Dashboard/scenes/ChangeState/index.tsx [hunk 1]
```diff
@@ -62,12 +63,18 @@ const programStates: State[] = [
   },
   {
     stepNo: 5,
-    name: 'Ongoing',
+    name: 'Mentee Confirmation',
```
Comment (anjula-sack) [inline src/scenes/Dashboard/scenes/ChangeState/index.tsx]: Isn't it Mentor Confirmation?

**Review by anjula-sack — CHANGES_REQUESTED** [review]

**Comment by Gravewalker666** [comment]
Hey, dude, @Gimhan-minion can you change this part
image
to this
image

File: src/scenes/Dashboard/scenes/ChangeState/index.tsx (hunk 1 above)
Comment (Gimhan-minion) [reply src/scenes/Dashboard/scenes/ChangeState/index.tsx]: My mistake. Thank you @anjula-sack fixed it.

**Comment by Gimhan-minion** [comment]
> Hey, dude, @Gimhan-minion can you change this part
> image
> to this
> image

@Gravewalker666 Fixed it! :)

**Review by anjula-sack — APPROVED** [review]
Thank you! @Gimhan-minion

**Pull request merged**